### PR TITLE
Add Swych Perpetual DEX adapter

### PR DESCRIPTION
**Pull request by itsmenikolaos**
The initial version of the Swych Perpetual DEX Adapter.

Features added:
- TVL
- Events

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/swych exports TVL: 
        
 
 
    bsc                       526.89 k
    
    total                    526.89 k

File: projects/swych/data/events.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const EVENTS = [
```
Comment (g1nt0ki) [inline projects/swych/data/events.js:1]: these are way too many events, do any of them have material impact on tvl chart? I would strongly recommend skipping those that dont

Comment (itsmenikolaos) [reply projects/swych/data/events.js:1]: Got you. Thanks for the info. We had no clue that this should be limited. How many events do you recommend displaying?

Comment (g1nt0ki) [reply projects/swych/data/events.js:1]: hi @itsmenikolaos ideally only the ones that affect the tvl chart. For example, dont think dashboard launch event has an impact.

I would say, restrict it to two for now :) 

Sorry about refactoring and merging here: https://github.com/DefiLlama/DefiLlama-Adapters/pull/9541 rest of your code is good, I just prefer everything in a single file, easier to read. Can you put up a new PR with 2 events you like to include?

**Pull request closed without merging**